components/signup-thunderbird.jsx

```diff
@@ -12,12 +12,8 @@ var Form = React.createClass({
       <div>
         <div className="container">
           <div className="wrap">
-            <div className="row base-line-thank-you">
```

Comment:
Some changes for the new-flow required markup, so we did this with two blocks, a base-line-file and a new-flow-file.

We can remove all base-line-file blocks and just use new-flow-file contents without the new-flow-file wrapper.

I do this a bunch in these changes.